utils/_context/_scenarios/__init__.py

```diff
@@ -751,6 +751,20 @@ def all_endtoend_scenarios(test_object):
         scenario_groups=[ScenarioGroup.DEBUGGER],
     )
 
+    debugger_exception_replay = EndToEndScenario(
+        "DEBUGGER_EXCEPTION_REPLAY",
+        rc_api_enabled=True,
+        weblog_env={
+            "DD_DYNAMIC_INSTRUMENTATION_ENABLED": "1",
```

Comment:
Nitpick: the env var of DI is not required for Exception Replay to function, I would even cover in a System Test. In the [Exception Replay testing PR](https://github.com/DataDog/dd-trace-dotnet/pull/5821) I test for with/without DI to make sure Exception Replay works regardless.

Reply:
Nice! I removed the env variable here e05180b43bd9289e536dd835711649b3297b68e3

Reply:
I've restored this env variable, since `java` doesn't work without it